bin/libtelco5g.py

```diff
@@ -570,10 +571,38 @@ def cache_cases(cfg):
 
   redis_set('cases', json.dumps(cases))
 
+def cache_bz(cfg):
+    
+    logging.warning("fetching cases")
+    cases = redis_get('cases')
+    bz_url = "bugzilla.redhat.com"
+    bz_api = bugzilla.Bugzilla(bz_url, api_key=cfg['bz_key'])
+    bz_dict = {}
+    token = get_token(cfg['offline_token'])
+    headers = {"Accept": "application/json", "Authorization": "Bearer " + token}
+
+    logging.warning("getting all bugzillas")
```

Comment:
Hmm... Right now we're getting `case_bugzillaNumber` when we cache the cases. But `bugzillas` is always there? Maybe you should just update `cache_cases` to pull it then, rather than iterate through the cases again for this.

Reply:
OK, should we get rid of the separate scheduler for the BZ info as well? [link](https://github.com/RHsyseng/t5g-field-support-team-utils/pull/55/files/24265ee9f6314d826bbea56993c80fdfc7d37ab4#diff-5d30f0c0c3d77c174e45663522134bc94ef37f62969d614ff49739335f4fa876R46)

Reply:
Nah, I think separate is still good. It still takes a bunch of extra minutes - and we want to be good api users - so once a day is enough. We need the main case data more often so we can create new cards quickly.